src/utils/colorScheme.tsx

```diff
@@ -1,3 +1,4 @@
+import { scaleOrdinal } from "d3-scale";
```

Comment:
Is this an unused var? I can see its been added but not where it has been used.

Reply:
Yes its unused, thought I needed it then I didn't! Thanks - updated